src/cascade/model/priors.py

```diff
@@ -213,3 +213,42 @@ def _parameters(self):
 ZERO = Uniform(0, 0, 0, name="constrain_to_zero")
 ZERO_TO_ONE = Uniform(0, 1, 0.1, name="uniform_zero_to_one")
 MINUS_ONE_TO_ONE = Uniform(-1, 1, 0, name="uniform_negative_one_to_one")
+
+
+DENSITY_ID_TO_PRIOR = {
```

Comment:
Yeah, I was wondering about that. I leaned towards a mapping because it was a relationship among existing items, not the definition of a _class_ of items. I'll stare at this again.